flax/errors.py

```diff
@@ -168,8 +168,9 @@ def __call__(self, inputs, embed_name='embedding'):
     _ = Embed().apply(variables, jnp.ones((5, 5, 1)), 'embed')
   """
   def __init__(self, param_name, scope_path):
-    super().__init__(f'No parameter named "{param_name}" exists in '
-                     f'"{scope_path}".')
+    super().__init__(
+        f'Could not find parameter named "{param_name}" in '
+        f'"{scope_path}". You may be passing an incorrect variable dict.')
```

Comment:
I'm not sure if "passing an incorrect variable dict" is very clear.

Maybe it would be better to add a second example to the error pydoc (that would then appear on the linked page https://flax.readthedocs.io/en/latest/flax.errors.html#flax.errors.ScopeParamNotFoundError )

@marcvanzee who proposed to add "Maybe you passing in an incorrect variable dict" on #1303

Reply:
Thanks. I find the new message clearer.